File: code/jvm/src/main/kotlin/pt/isel/leic/ptgest/domain/workout/model/ExerciseDetails.kt
package pt.isel.leic.ptgest.domain.workout.model

import pt.isel.leic.ptgest.domain.workout.Modality
import pt.isel.leic.ptgest.domain.workout.MuscleGroup

data class ExerciseDetails(
    val id: Int,
    val name: String,
    val description: String?,
    val muscleGroup: List<MuscleGroup>,
    val modality: Modality,
    val ref: String?
)

File: code/js/src/views/user/TrainerViews/models/exerciseDetails.ts
class ExerciseDetails {

    id : string;
    name : string;
    description : string;
    muscleGroup : string[];
    modality : string;
    ref : string;
    constructor(id: string, name: string, description: string, muscleGroup: string[], modality: string, ref: string) {
        this.id = id;
        this.name = name;
        this.description = description;
        this.muscleGroup = muscleGroup;
        this.modality = modality;
        this.ref = ref;
    }

}
